### ISSUE TITLE
Migrate Hub Metrics to New AWS Data Account

### ISSUE THREAD

**Issue by pnadolny13** [meltano/squared#247]
Currently were using the legacy AWS account for transferring hub metrics files. We should update this to use the new data AWS account and make a new system user that has strict access and only manages this workflow.

**Comment by aaronsteers** [comment]
@pnadolny13 - Per a recent comment, I presume this is done already. Can you confirm if metrics are all running in the Data account or if we have anything running in the main/original AWS account?

**Comment by pnadolny13** [comment]
@aaronsteers no this isnt done yet, the S3 bucket that we use for the hub is in the legacy account. I dont think anything else is really running there, theres data in S3/Athena but I think we could probably just drop that at this point, maybe we can do a final audit before doing that.

**Comment by pnadolny13** [comment]
- create squared user with write access on bucket prefix
- create hub user with read access on bucket prefix
- update squared aws-cli job to write to that prefix
- test
- PR to hub with bucket change and note that build credentials need to be updated
- Once hub PR is ready to merge, update squared, run job once, then update hub creds and merge
-

**Comment by pnadolny13** [comment]
The new plan is to wait until https://github.com/meltano/infra/issues/254#issuecomment-1369860911 is done, then use that S3 bucket.

**Comment by magreenbaum** [comment]
Bucket name is: `hub-metrics-b8ba2d5`. New squared user now has write access on this bucket. 
Hub will access via new user in 1pass `Hub Metrics Bucket Read Access`. We'll need to update hub repo and Netlify with new bucket/creds when the time comes.

**Comment by pnadolny13** [comment]
These are the things I'll plan to do to migrate:

Squared:
- update configs to point to the new bucket/prefix
- update CI creds to use the new creds. Right now we have a dev and prod bucket for Squared, should I just use 2 distinct prefixes or is there a dev bucket I can write to?
- update the self hosted AWS SSM secrets to use the new creds
- update the repo secrets.yml for cloud to use the new creds

Hub:
- update path to read S3 files from https://github.com/meltano/hub/blob/a0b2bb3e039d15cc7656298c42f4a422f5c437fa/utility_scripts/build/build_with_metrics.sh#L23
- update github actions env vars to use new read access hub user creds

**Comment by magreenbaum** [comment]
@pnadolny13 I didn't realize there was a dev and prod separate bucket 😬. We do have a staging environment for the squared/hub user but that would require using separate creds. If there are no concerns around the two data sets being in the same bucket, using separate prefixes seems fine to me. If not, we can definitely use the staging env for dev stuff.

**Comment by pnadolny13** [comment]
@magreenbaum to be honest I didnt realize that either until I was making that todo list above 😞 . I think separate prefixes is a fine solution for now. Eventually once its easier to manage multiple sets of credentials (i.e. distinct secrets.yml per environment?) I would prefer to disconnect these but totally fine for now.
